tests/v2/validation/pipeline/releaseupgrade/main.go

```diff
@@ -104,7 +106,16 @@ func main() {
 	clusters := new(pipeline.Clusters)
 	config.LoadConfig(pipeline.ClustersConfigKey, clusters)
 
-	for i, v := range clusters.RKE1Clusters.CustomClusters {
+	local := clusters.Local
+	if local != nil {
+		newConfigName := config.NewConfigFileName(dirName, localFileName)
+		err := NewRancherLocalClusterConfiguration(*local, newConfigName, copiedConfig)
+		if err != nil {
+			logrus.Info("error while generating a rancher cluster config", err)
+		}
+	}
```

Comment:
should this fail if the local cluster returns nil?

Reply:
Great question, we don't know which pipeline is using this package. So we can't really bind this to a local cluster check. However, on each pipeline, we will see this log. This is to increase visibility, especially on the local upgrades pipeline.